dear_petition/petition/export/main.py

```diff
@@ -9,13 +9,18 @@
 
 
 FORM_TYPE_MAP = {
+    constants.DATA_PETITION: DataPetitionForm,
     constants.DISMISSED: AOCFormCR287,
     constants.ATTACHMENT: AOCFormCR285,
 }
 
 
 def build_pdf_template_context(petition, extra):
-    Form = FORM_TYPE_MAP.get(petition.form_type, AOCFormCR287)
+    if hasattr(petition, "data_only") and petition.data_only:
+        form_type = constants.DATA_PETITION
+    else:
+        form_type = petition.form_type
+    Form = FORM_TYPE_MAP.get(form_type, AOCFormCR287)
```

Comment:
Should there be a default for `form_type`? I feel like it should be an error if we pass in a `form_type` that is not found in the dictionary.

Reply:
``form_type`` is controlled by constants, but it's possible a developer may introduce an issue down the road. We'd need to rethink the flow a bit, since at this point we're trusting the data to be valid and don't currently have a mechanism in place to catch the error earlier in the call stack.